sqswatcher/send_sqs.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+
+cfn_region=us-east-1
+cfn_sqs_queue=https://sqs.us-east-1.amazonaws.com/822857487308/AS-SQS
+cfn_instance_slots=3
+instance_id=i-04fcafee570cd7073
```

Comment:
What is this `instance_id`? Is this file a sample?

Reply:
Removed this file. It was just used for testing.